README.md

```diff
@@ -171,6 +171,7 @@ Command flag              | Description
 `-C/--cache`              | Store and reuse results for faster operation.
 `-d/--debug`              | Displays some extra debug output.
 `-D/--display-cop-names`  | Displays cop names in offense messages.
+`-e/--extra-details`      | Displays extra details in offense messages.
```

Comment:
I'd use `E` or `X` for this. I reserve lowercase switches for flags that are likely to be used very often.